packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet.ts

```diff
@@ -153,7 +155,7 @@ export class Wallet implements IReadWriteWallet {
 
 		return container
 			.get<IExchangeRateService>(Identifiers.ExchangeRateService)
-			.exchange(this.currency(), this.exchangeCurrency(), DateTime.make(), this.balance().divide(1e8));
+			.exchange(this.currency(), this.exchangeCurrency(), DateTime.make(), this.balance().divide(this.#denomination));
```

Comment:
```suggestion
			.exchange(this.currency(), this.exchangeCurrency(), DateTime.make(), this.balance().toHuman());
```

Shouldn't this yield the same?

Reply:
No the method return a `BigNumber`, toHuman is a `string`

Reply:
Would it be cleaner to introduce a new method on `BigNumber` that returns what `toString()` does but as a `BigNumber`? Then we don't need to manually call this `divide`.

Reply:
Yea good point. Let me check all places where it could be used